installation/linux.md

```diff
@@ -775,8 +775,8 @@ You can also manually connect:
 - **On Mac OS X:** Open Finder → Go → Connect to Server: `smb://openhab@openhab-device.local`
 - **On Windows:** Open Windows Explorer → Address bar: `\\openhab-device.local` → Right click a share and assign a drive letter
 
-Be sure to use the actual host name instead of `openhab-device`.
-When asked, authenticate with the username "openhab" and the chosen password.
+Be sure to use the actual host name instead of `openhab-device.local`, the default host name is `openhabianpi`.
```

Comment:
The `local` depends on your network infrastructure i think.
For me it would be `openhab-device.fritz.box`, so i dont think it is a good idea to add a specific domain ending over there.